cvat/apps/engine/static/engine/js/shapeFilter.js

```diff
@@ -26,6 +26,13 @@ class FilterModel {
             lock: shape.model.lock
         };
 
+        if (shape.model.type.split('_')[1] === 'box') {
```

Comment:
I see that all other shapes will be filtered out if specify `*[width > 1]`. Does it make sense? Probably it is better to estimate the bounding box for polygons, polylines, points and put here width and height of the bounding box.

Reply:
We can do it, but there weren't any requests to estimate size of polyshapes. 
Also it requires more computations because we need manually compute width and height of bounding box in the model.

Reply:
Done